.github/workflows/push_code_linting.yml

```
@@ -32,19 +32,19 @@ jobs:
       NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
       GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
     steps:
-      - uses: actions/checkout@v4.1.1
+      - uses: actions/checkout@v4.2.2
 
       - name: Install compatible Nodejs version
         id: setup-node
         uses: ./.github/actions/setup-node
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)